### PR TITLE
Implement emergency functions component

### PR DESCRIPTION
**Pull request by Mloweedgar**
ref https://github.com/CodeTanzania/ewea/issues/16

### CODE REVIEW

**Comment by BenMaruchu** [comment]
@Mloweedgar  Have view reverted that changes yet?

**Comment by Mloweedgar** [comment]
@BenMaruchu I have reverted the changes I made in searchableSelectInput Componet

File: src/util.js [hunk 1]
```diff
@@ -118,3 +118,36 @@ export const getRGBAColor = (baseColor, alpha) => {
...
+ * @function
+ * @name isString
+ * @description checks if a variable is a string
+ * @param {string} data  string to truncate
+ * @returns {boolean} truthy value based on comparison
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+export const isString = data =>
+  typeof data === 'string' || data instanceof String;
```
Comment (BenMaruchu) [inline src/util.js]: @Mloweedgar Hello, Can we use lodash for utility function like this to avoid reinventing the wheel?

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/util.js (hunk 1 above)
Comment (Mloweedgar) [reply src/util.js]: sure we can

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**